### ISSUE TITLE
Pipewire device selection isn't actually a device

### ISSUE THREAD

**Issue by ProjectPatatoe** [mumble-voip/mumble#5810]
### Description

Mumble Pipewire output/input is not giving option to select device and instead shows the channels (Mono/Stereo). It seems to be automatically selecting the first device on the recording list.

### Steps to reproduce

1. Select Pipewire in Input->System
2. Select Pipewire in Output->System

### Mumble version

1.4.230

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

This is on Steam Deck. SteamOS (Arch,kde)

I don't have pipewire on any other system so I don't know if this a SteamOS thing or a Pipewire thing

### Screenshots

![Screenshot_20220818_203942](https://user-images.githubusercontent.com/27925441/185538901-69514fab-0d55-41eb-965c-40ee188c6630.png)
---
![Screenshot_20220818_205510](https://user-images.githubusercontent.com/27925441/185540145-9474f2f8-7a22-4160-a277-cb4e4be9f9b4.png)
---
![Screenshot_20220818_205536](https://user-images.githubusercontent.com/27925441/185540156-bc833fde-9f31-4955-acdd-11d897f269ce.png)

**Comment by davidebeatrici** [comment]
This is specific to our implementation, I'll add a proper device selector.

Cool to see Mumble running on a Steam Deck!

**Comment by ProjectPatatoe** [comment]
I realized I forgot to mention, switching to Pulseaudio is a workaround for anyone else looking to get this working without messing around with tricking the device order and the such.

**Comment by davidebeatrici** [comment]
The functionality is implemented in libcrossaudio, which will be used in Mumble to replace our current audio backend implementations.

**Comment by DominicWrege** [comment]
Any updates on this ? Do we have a workaround?

**Comment by steveyh25** [comment]
> Any updates on this ? Do we have a workaround?

yes... it has already been mentioned to select PulseAudio instead
